base/ltxref.dtx

```diff
@@ -87,6 +89,10 @@
 %  leaving a space on both sides of it is equivalent to leaving
 %  a space on either side.
 %
+%  Note: the starred version |\ref*| and |\pageref*| are provided
+%  to align with the use of hyperref. Without \pkg{hyperref} (or some
+%  other package using the starred form) it is simply ignored.
```

Comment:
Imho, the wording here is a bit misleading, I'd change it to `the star is simply ignored` instead of "it".